src/base/geom/circle.h

```diff
@@ -2,9 +2,11 @@
 #define GEOM_CIRCLE
 
 #include <optional>
+#include <array>
```

Comment:
Use this include only at solutions header

Reply:
done